### DISCUSSION TITLE
How do I use a single page multi table builder?

### DISCUSSION THREAD

**Discussion by ibrahim-bedir** [filamentphp/filament#6943] [Help] [answered]
I installed table builder in my project in the current livewire broadcast, but there is more than one table on the page, how can it do this?

You can think with the logic of multiple form builder.
`

            'postForm' => $this->makeForm()
                ->schema($this->getPostFormSchema())
                ->model($this->post),
            'authorForm' => $this->makeForm()
                ->schema($this->getAuthorFormSchema())
                ->model($this->author),
`

**Comment by danharrin** [comment, marked as answer]
You can nest multiple Livewire components, each using the table builder, on one page.

**Reply by ibrahim-bedir** [reply]
thank you
